tests/framework/installer/ceph_installer.go

```diff
@@ -41,7 +41,7 @@ import (
 
 const (
 	// test with the latest nautilus build
-	nautilusTestImage = "ceph/ceph:v14.2.12"
+	nautilusTestImage = "ceph/ceph:v14"
```

Comment:
`14.2.14` did not release with the fix so we have 2 broken releases now. We need to stick with 14.2.12...